main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,166 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import ca.uwaterloo.flix.language.ast.{SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.util.InternalCompilerException
+import ca.uwaterloo.flix.util.collection.Bimap
+
+import scala.collection.immutable.SortedSet
+import org.sosy_lab.pjbdd.api.{Builders, DD}
+
 object BddFormula {
 
-  class Bdd
+  val creator = Builders.bddBuilder().build()
```

Comment:
private-- making everything private when possible :)

Reply:
Since this is a constant, Scala conventions would call it `Creater`, or even better `BddBuilder` or something like that.

Reply:
Even better: `GlobalBddBuilder` with a comment that this a factory for creating BDDs and that it is thread-safe (I hope!).